components/brave_new_tab_ui/containers/newTab/gridSites.tsx

```diff
@@ -18,17 +18,26 @@ import createWidget from '../../components/default/widget'
 import { MAX_GRID_SIZE } from '../../constants/new_tab_ui'
 import { useMaintainScrollPosition } from '../../helpers/scrolling'
 import AddSiteTile from './addSiteTile'
-import { GridPageButtons } from './gridPageButtons'
 // Component groups
 import GridSiteTile from './gridTile'
 import { TopSiteDragOverlay } from './gridTileOverlay'
+import Navdots from '@brave/leo/react/navdots'
+import styled from 'styled-components'
 
 // Note: to increase the number of pages, you will also need to increase
 // kMaxNumCustomLinks in ntp_tiles/constants.cc
 const MAX_PAGES = 4
 const activationConstraint: PointerActivationConstraint = { distance: 2 }
 const autoScrollOptions: AutoScrollOptions = { interval: 500 }
 
+const NavDotsContainer = styled.div`
+  --leo-navdots-active-color: white;
+  --leo-navdots-active-color-hover: white;
+  --leo-navdots-color: rgba(255,255,255,0.5);
+  --leo-navdots-color-hover: rgba(255,255,255,0.8);
+  --leo-navdots-transition-duration: 0;
+`
+
```

Comment:
Hmmm good question, I'll have a play and see!

Reply:
Okay, so it doesn't but not because it can't but because we haven't wired it up. Basically, we aren't passing the `className` prop through to the web-component.

I'm going to leave this as-is for now, but I'm also going to update the React wrapper to allow setting a few common props on the web-component (because I think we should definitely support this use case).

Reply:
See https://github.com/brave/leo/pull/192

Reply:
Okay, as https://github.com/brave/leo/pull/192 landed I updated this to just style the component :smile: